src/storybook-components/_data/demoData.js

```diff
@@ -1241,6 +1233,51 @@ const demoData = {
             placeholder: 'Enter a date'
           },
 
+          {
+            type: 'rpldivider'
+          },
+
+          {
+            type: 'rplcheckbox',
+            inlineLabel: 'Check box selected',
+            model: 'checkbox',
+            default: true
+          },
+
+          {
+            type: 'rplchecklist',
+            label: 'Multi-select list box',
+            model: 'multichecklist',
+            listBox: true,
+            hint: 'Implemented using rplchecklist with listBox: true',
+            placeholder: 'Select multiple topics',
+            values: ['Topic A', 'Topic B', 'Topic C', 'Topic D']
+          },
+
+          {
+            type: 'rplchecklist',
+            label: 'Multi-select drop down',
+            model: 'multichecklist',
+            hint: 'Implemented using rplchecklist',
+            placeholder: 'Select multiple topics',
+            values: ['Topic A', 'Topic B', 'Topic C', 'Topic D']
+          },
+
+          {
+            type: 'vueMultiSelect',
+            model: 'singlechecklist',
```

Comment:
Though it says type is `vueMultiSelect ` but it's using `singlechecklist`?

Reply:
Updated naming of model to relate to vuemultiselect.